bytecomp/bytelink.ml

```diff
@@ -536,8 +536,13 @@ let link_bytecode_as_c ppf tolink outfile =
 
 let build_custom_runtime prim_name exec_name =
   let runtime_lib = "-lcamlrun" ^ !Clflags.runtime_variant in
+  let debug_prefix_map =
+    if Config.c_has_debug_prefix_map then
```

Comment:
We should probably not pass this option if `-dcamlprimc` is passed as well.

Reply:
I realized that my definition of "reproducible" was slightly
broken: I understood it as "produces the same contents"
while it is "produces the contents in the same file".